modules/nw-operator-vsphere-multiple-nic.adoc

```diff
@@ -0,0 +1,86 @@
+
+[IMPORTANT]
+====
+If you configured multiple NICs for a network implementation but intend to use only a single NIC for your current needs, ensure that you specify the same IP address for the internal and external CIDR implementations. This configuration ensures that the CCM selects your preferred NIC and not the first NIC that the host operating system reports.
+====
+
+.Example error message because of the {vmw-short} CCM selecting an incompatible NIC
+[source, text]
+----
+ERROR Bootstrap failed to complete: timed out waiting for the condition
+ERROR Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane.
+----
+
+.Prerequisites
+
+* You created Kubernetes manifest files for your {product-title} cluster.
+
+.Procedure
+
+. Enter the following command in the {oc-first} tool to view detailed information, especially `Internal-IP` or `External-IP information, for each node that exists in your cluster. You will need this information in a the `nodeNetworking:` section of the `nodeNetworking` object.
++
+[source,terminal]
+----
+$ oc get nodes -o wide
+----
+
+. From the directory where you store your {product-title} cluster manifest files, open the `manifests/cluster-infrastructure-02-config.yml` manifest file.
+
+. Add a `nodeNetworking` object to the file and specify internal and external network subnet CIDR implementations for the object.
++
+.Example of a configured `cluster-infrastructure-02-config.yml` manifest file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+  name: cluster
+spec:
+  cloudConfig:
+    key: config
+    name: cloud-provider-config
+  platformSpec:
+    type: VSphere
+    vsphere:
+      failureDomains:
+      - name: generated-failure-domain
+      ...
+       nodeNetworking:
+         external: <1>
+           networkSubnetCidr:
+           - <machine_network_cidr_ipv4>
+           - <machine_network_cidr_ipv6>
+         internal: <2>
+           networkSubnetCidr:
+           - <machine_network_cidr_ipv4>
+           - <machine_network_cidr_ipv6>
+      vcenters:
+      - datacenters:
+        - sample-dc
+        port: 443
+        server: vcenter.example.com
+status:
+  ...
+  platformStatus:
+    type: VSphere
+# ...
+----
+<1> The IP address of a node that exists outside to your cluster. The CIDR must match the CIDR that you specified in the `networking.machineNetwork` section of your `install-config.yaml` file.
```

Comment:
I don't know that it will be true that this CIDR will appear in machineNetworks. In my experience those are generally not an exhaustive list of every network a node will have, just the one Kubelet will use.

Also, these are CIDRs not IPs. And I don't see anything in the original KCS to indicate that this needs to be an actual node address.

I wonder if we just want to link to the [VSphere API docs](https://docs.openshift.com/container-platform/4.14/rest_api/config_apis/infrastructure-config-openshift-io-v1.html#spec-platformspec-vsphere-nodenetworking) for these fields because we're duplicating content here and we're also not documenting all the ways these values can be provided. If the API docs aren't clear enough* then we should fix that.

*: Which may well be the case. Without my own foreknowledge of how this is supposed to work I'm not sure I could figure out which CIDR should go where.

Reply:
Good idea. Links already in place so I dropped the descriptions.